quiz_verification.php

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+		<meta name="author" content="Jackson Alexander">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="my_style.css">
+		
+		<style>
+			.image-container{
+				float: left;
+				height: 20%;
+				width: 30%;
+			}
+			
+			.image{
+				width: 100%;
+				height: 100%;
+			}
+			
+			.selected{
+				border: 5px solid purple;
+			}
+		
+		
+		</style>
+		
+</head>
+
+<body style="background-color:#383838;">
+
+<?php
+
+include 'nav.html';
```

Comment:
For a better MVC style, you could make all the verifications below first, at the top of the page, and remember your warning message. Ex:

```
$msg = false;
if($input_length != 7){
	$msg =  "make sure all fields in the form have been filled out" . $input_length;
}
...
```

Then at the bottom:
```
<! ---------------VIEW SECTION------------------------------->
<! DOCTYPE html>
<?php include 'nav.html'; ?>
....
```

It would also be easier to read.